### PR TITLE
Updates to documentation text, URLs and build config

### PR DESCRIPTION
**Pull request by genodeftest**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -32,31 +33,43 @@
...
-source_encoding = 'utf-8'
-
 # The master toctree document.
 master_doc = 'index'
 
 # General information about the project.
 project = u'Exaile'
-copyright = u'2009-2014 Exaile Development Team'
+copyright = u'2009-2017 Exaile Development Team'
+author = u'Adam Olsen <arolsen@gmail.com>, Johannes Sasongko <sasongko@gmail.com>, Aren Olsen <reacocard@gmail.com>, Mathias Brodala <info@noctus.net>, Christian Stadelmann'
```
Comment (virtuald) [inline doc/conf.py]: Forgot me. :(

Comment (genodeftest) [inline doc/conf.py]: Oops, I expected the document to be so old, that you were included already… (copy-pasted from below, where the same mistake is, a second time)

Comment (genodeftest) [inline doc/conf.py]: @virtuald : Sorry…. Now I also added your Email address, which is in some of the source files.

Comment (virtuald) [inline doc/conf.py]: Haha, no worries.

**Pull request merged**